synchronizer/store.go

```diff
@@ -10,18 +10,23 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-const dbTimeout = 2 * time.Second
+// SyncTask is the type of the sync task
+type SyncTask string
```

Comment:
What is the benefit of having this type alias (as oposing to working with the `string` type)?

In the end, I don't even understand why function calls are parameterizable and receive `SyncTask`, since `L1SyncTask` const is always used?

Reply:
The idea was to be able to support many task types but there is only one for now. A custom type makes what kind of values can be passed to the function more explicit. Even if there is only one value for now, this approach looks more accurate.

Reply:
I understand, but why is it a string and not an enumeration? :slightly_smiling_face:

Reply:
Defined as a string in the DB so just following the current DB structure.

Reply:
As you know, you can provide the `String` function and this is obviously abuse of the string type... But ok I don't want to push any further, it is pointless.

Feel free to merge the PR.